### ISSUE TITLE
Identity Management FAQ

### ISSUE THREAD

**Issue by kchristidis** [openblockchain/obc-docs#136]
[Link](https://github.com/openblockchain/obc-docs/blob/master/FAQ/identity_management_FAQ.md):

> Would its membership service make OBC a centralized solution?

It is not clear to me what the answer is here. Can we consider rewording it?

**Comment by frankyclu** [comment]
https://github.com/openblockchain/obc-docs/blob/master/FAQ/identity_management_FAQ.md

check the updated version

**Comment by kchristidis** [comment]
As we discussed earlier today, I think additional revising is needed.

Here's a much more balanced answer, also influenced by a discussion I had with Chet on the matter:

> The membership service does centralize membership. (There is no other way around it. In a properly set-up permissioned network, members should be vetted.) This does not however imply anything about the content or the ordering of the transactions in the network.

If I were a third-party developer, when reading the "Would its membership service make OBC a centralized solution?" question, I'd expect to see a "yes" somewhere in there.

And I'd be alarmed if I didn't see one.

That is the case with the current answer, which cuts straight to the last sentence of my suggested response.

I therefore argue that we should either revise the question (if we just want to highlight the second part), or keep the question but modify the answer (to include the first past).

**Comment by joshhus** [comment]
Agree, maybe the answer is: "Although OBC is a decentralized solution for transactions, its membership service (as a permissioned network) is centralized." 
This is assuming that the question makes and important distinction -- and is one that customers would ask.

**Comment by kchristidis** [comment]
I like @joshhus's version as well. Not sure why the answer is not more straightforward.

**Comment by quiquee** [comment]
Even better, being this a Identity Management FAQ I would avoid in this section questions that refer to the centralized or decentralized architecture of OBC itself. I would rephrase the question to "Is authority management centralised" and the answer is "Yes , it is"
I think the whole point of those, like me, interested in OBC and Hyperledger is that it can be used by banks, and banks require its clients to delegate their identity check to the institution they sign the contract with. There is nothing wrong with that so I would prefer to have a clear answer so people can make the difference.

**Comment by binhn** [comment]
@quiquee +1

**Comment by kchristidis** [comment]
@quiquee Completely agree.
